translations/en.json

```diff
@@ -548,7 +548,6 @@
     "unknown-realm-error": "Something went wrong",
     "notifications": "Notifications",
     "currency-display-name-czk": "Czech Koruna",
-    "extension-installed": "You installed an extension",
```

Comment:
just wondering how is this change related to this PR?

Reply:
'cose in the PR I removed the alerts related to the installation\uninstallation which overlap with extension screen in case the extension uses lifecycle events to show a welcome\splash (see https://github.com/status-im/status-react/pull/7979#issuecomment-484029746). maybe alerts in lifecycle events could become a default bestpractice in extension developement? (we loose translation)